requirements/dev.txt

```diff
@@ -132,6 +135,7 @@ pyhamcrest==1.9.0         # via twisted
 pyjwt==1.7.1
 pyldap==3.0.0.post1       # via fakeldap
 pylibmc==1.6.0
+pynacl==1.3.0             # via paramiko
 pyoembed==0.1.2
 pyopenssl==19.0.0         # via ndg-httpsclient, requests, scrapy
 pyparsing==2.3.1          # via packaging
```

Comment:
We might want to use a separate `requirements.txt` for this project, assuming I'm correct that it doesn't need to share code with any of the rest of Zulip?